internal/store/installation_test.go

```diff
@@ -1292,3 +1309,7 @@ func fixDNSRecords(num int) []*model.InstallationDNS {
 		{DomainName: fmt.Sprintf("dns-%d.example.com", num)},
 	}
 }
+
+func bToP(b bool) *bool {
+	return &b
+}
```

Comment:
Can you use the one you added [here](https://github.com/mattermost/mattermost-cloud/pull/974/files#diff-8c3c91c8947fcc1b9fbfc09a56bb021b291aea7d6432a2ba751c8d93cec0c690R76)?

Reply:
I actually can't cleanly because that one belongs to the `api_test` package. I am going to open a ticket to clean all this stuff up. We have a lot more of these hanging around simply because we didn't want to import large packages for simple test tooling.